app/models/AddressLookup.scala

```diff
@@ -56,6 +56,18 @@ case class AddressLookup(addressLine1: Option[String],
       country.map(_.description)
     ).flatten.mkString(", ")
 
+  lazy val formatRadios: String =
+    Seq(
+      addressLine1,
+      addressLine2,
+      addressLine3,
+      addressLine4,
+      Option(town),
+      Option(postcode),
+      county,
+      country.filterNot(_.description == "United Kingdom").map(_.description)
+    ).flatten.mkString(", ")
```

Comment:
I don't think we need a new `formatRadios`, but can simply filter it in the already present `format`, as we must not show country (if UK) anywhere I believe (as far I remember from previous discussion).
Also the filter should be based on the country code and not the description:
```
  lazy val format: String =
    Seq(
      addressLine1,
      addressLine2,
      addressLine3,
      addressLine4,
      Option(town),
      Option(postcode),
      county,
      country.filterNot(_.code == "GB").map(_.description)
    ).flatten.mkString(", ")
    ```